test/provision/k8s_install.sh

```diff
@@ -330,6 +331,20 @@ case $K8S_VERSION in
         CONTROLLER_FEATURE_GATES="EndpointSlice=true"
         API_SERVER_FEATURE_GATES="EndpointSlice=true"
         ;;
+    "1.20")
+        # kubeadm 1.19 requires conntrack to be installed, we can remove this
```

Comment:
Nit: Should this be `kubeadm 1.20`?

Reply:
I'll fix it up once K8s 1.20 is released, which is tomorrow.